HerPublicWebsite.BusinessLogic/ExternalServices/EmailSending/GovUkNotifyConfiguration.cs

```diff
@@ -7,6 +7,7 @@ public class GovUkNotifyConfiguration
     public string ApiKey { get; set; }
     public string ComplianceEmailRecipients { get; set; }
     public ReferenceCodeConfiguration ReferenceCodeTemplate { get; set; }
+    public ReferenceCodeConfiguration ReferenceCodePendingLATemplate { get; set; }
```

Comment:
Let's call these `ReferenceCodeForLiveLocalAuthorityTemplate` and `ReferenceCodeForPendingLocalAuthorityTemplate`.

Two word abbreviations always look a bit awkward in names!